section/session-flow.html

```diff
@@ -321,5 +334,29 @@ <h3>Payment streaming</h3>
       </ol>
       <p class="note">For example, <code>convert scaled amount to decimal(100, 2)</code> returns <code>"1.00"</code>.</p>
     </div>
+
+    <div class="algorithm">
+      <p>To <dfn>revoke user wallet credentials</dfn>, run these steps:</p>
+      <ol>
+        <li>Run [=update grant access token=] with the empty string.</li>
+        <li>For each |session| in the [=active sessions list=]:
```

Comment:
Clarifying question: is `active sessions list` per document or single list of entire browser? Because if per document/tab, we need to stop all sessions - even for other tabs.